classes/fields/time.php

```diff
@@ -150,7 +150,7 @@ public function schema ( $options = null ) {
     public function display ( $value = null, $name = null, $options = null, $pod = null, $id = null ) {
         $format = $this->format( $options );
 
-        if ( !empty( $value ) && !in_array( $value, array( '0000-00-00', '0000-00-00 00:00:00', '00:00:00' ) ) ) {
```

Comment:
I suspect the code was just copied from datetime.php initially and that came along.  It wouldn't hurt to have the empty dates in but they should never be seen by this code